opencensus/exporters/stats/stackdriver/internal/stackdriver_utils.cc

```diff
@@ -153,15 +149,28 @@ std::vector<google::monitoring::v3::TimeSeries> DataToTimeSeries(
 
 }  // namespace
 
+std::string MakeType(absl::string_view metric_name_prefix,
+                     absl::string_view view_name) {
+  return absl::StrCat(metric_name_prefix, view_name);
+}
+
+bool IsBuiltinMetric(absl::string_view metric_type) {
```

Comment:
honestly this looks very strange since this is a negative check against known custom metrics. That is not the opposite of builtin metrics, since there is no knowledge of identifying those.

Reply:
I also think it's a bit odd but it's what the golang version does. It's documented as "heuristically known"

Reply:
A repeated failure doesn't make a good feature. That said, this could at least be named: IsKnownCustomMetric.

Reply:
Done.